src/emm/emm_cli.py

```diff
@@ -211,5 +219,24 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_content_width=100))
+@click.option("--commit-message", default=None, help="Commit message to apply.")
```

Comment:
I think this should be a required field (`required=True`), then you won't need to include a default.